CONTRIBUTE.md

```diff
@@ -0,0 +1,66 @@
+# Distributed workflow documentation
+
+## Branching model
+For now we have these branches:
+* master
+* development
+* feature/\<feature-name\>
+* hotfix/\<fix-name\>
+* document/\<documentation-name\>
+
+Master is the release branch. Merge to master only from development. Merging needs to be done by a pull request.
+
+Development is the development branch.
+
+Feature branches are short-lived branches that branch off of development and get merged back to development. Merging needs to be done by a pull request. Document branches are the same but for documentation.
+
+Hotfix branches are used to correct bugs on master and development.
```

Comment:
I think we should only do hotfix branches from development. Then merge development into master and release the "hotfixed" version.

Reply:
I see. Good point, changeing it.

Reply:
Actually, on second thought, then you would also be including any changes made to development since the last deploy? Isn't it better to branch of master, fix bug, and then release only that as a patch? Then you can also merge that release into develop, meaning make the bug fix also on develop, and continue developing.

I guess if you're making deployments daily then it comes to the same thing, but what do you think? It's an interesting point.